lib/SILOptimizer/Mandatory/TFConstExpr.cpp

```diff
@@ -1550,7 +1569,7 @@ bool ConstExprEvaluator::decodeAllocUninitializedArray(
   bool hadUnknownUsers = false;
 
   // The call has a tuple return type, see if we can walk all uses of them to
-  // analyze them and find the stores to the elements.
+  // analyze them and find the stores/apply_insts to the elements.
```

Comment:
Nit: Replace "apply_insts" with just "apply's" here in comments? The use of snake case seems to refer to an instruction called `apply_inst` but it's actually `apply` :)

Same for other occurrences of "apply_inst".